etc/pipeline-build/Makefile

```diff
@@ -1,7 +0,0 @@
-.PHONY: docker-build push-to-minikube
-
-docker-build:
-	ls | grep -v Makefile | xargs -I {} docker build -t pachyderm/{}-build:$(shell $(GOPATH)/bin/pachctl version --client-only) {}
-
-push-to-minikube: docker-build
-	ls | grep -v Makefile | xargs -I {} ../kube/push-to-minikube.sh pachyderm/{}-build:$(shell $(GOPATH)/bin/pachctl version --client-only)
```

Comment:
I think you will need to call in the top makefile. Maybe create a target push-build-pipeline-to-minikube

`./etc/kube/push-to-minikube.sh pachyderm/go-build:$(VERSION)`
`./etc/kube/push-to-minikube.sh pachyderm/python-build:$(VERSION)`

Reply:
Done